### PR TITLE
Update roadmap now we're three months in

### PR DESCRIPTION
**Pull request by bethaitman**
First stab at what should be on the roadmap for the next three months. I've only had a very quick go at descriptions, so feel free to suggest better ones. And let me know if there's anything important that should be added.

### CODE REVIEW

File: docs/source/support/roadmap.rst [hunk 1]
```diff
@@ -1,63 +1,48 @@
...
-
-  `More about this on GitHub <https://github.com/digital-asset/daml/issues/125>`__.
-- **Web IDE**
-
-  Provide a browser-based version of :doc:`DAML Studio </daml/daml-studio>` to make it easier to try DAML out.
-
-  `More about this on GitHub <https://github.com/digital-asset/daml/issues/118>`__.
-- **DAML-on-X self-service package**
+  `Read more on GitHub <https://github.com/digital-asset/daml-js>`__.
+- **DAML-on-X self-service package: Beta release**
```
Comment (meiersi) [inline docs/source/support/roadmap.rst]: We might have misunderstood each other. Didn't know about the June scope for the roadmap. Our BETA release will be ready by mid August. Can you add a `BETA release by September` ?

Comment (meiersi) [reply docs/source/support/roadmap.rst]: @bethaitman : ignore. I've misunderstood the meaning of the date. Sorry for not reading it well.

Comment (bethaitman) [reply docs/source/support/roadmap.rst]: Yep - this is "last updated June 2019", no real delivery dates here!

File: docs/source/support/roadmap.rst [hunk 2]
```diff
@@ -1,63 +1,48 @@
...
 
 This page specifies the major features we’re planning to add next to the DAML SDK. Plans and timelines are subject to change. If you need any of these features or want to request others, see the :doc:`/support/support` page for how to get in touch.
 
 We plan to update this roadmap roughly every three months.
 
-- **Windows support**
+- **Generic templates**
 
-  Support for installing and running the SDK on Windows.
+  Support for writing generic templates - for example, a generic ``Proposal`` template that could be a ``Proposal<Iou>``.
```
Comment (hurryabit) [inline docs/source/support/roadmap.rst]:
```suggestion
  Support for writing generic templates - for example, a generic ``Proposal`` workflow that could be instantiated as a ``Proposal Iou`` to propose an ``Iou``.
```

**Comment by bethaitman** [comment]
@bitonic, is this ok with you from the Runtime side?

**Pull request merged**